android/java/org/chromium/chrome/browser/settings/BraveSyncScreensPreference.java

```diff
@@ -791,18 +809,24 @@ private enum SyncInputType { NEW, JOIN }
     private void seedWordsReceivedImpl(String seedWords, SyncInputType syncInputType) {
         pauseSyncStateChangedObserver();
 
-        getBraveSyncWorker().setJoinSyncChainCallback((Boolean result) -> {
-            if (result) {
-                if (syncInputType == SyncInputType.JOIN) {
-                    setAppropriateView();
-                }
-            } else {
-                // TODO(AlexeyBarabash): consider to have error code if there will
-                // be more than only one case of failure
-                onSyncError(getResources().getString(R.string.brave_sync_joining_deleted_account));
-                AllowNewQrScan();
-            }
-        });
+        getBraveSyncWorker()
+                .setJoinSyncChainCallback(
+                        (Boolean result) -> {
+                            if (result) {
+                                if (syncInputType == SyncInputType.JOIN) {
+                                    setAppropriateView();
+                                }
+                            } else {
+                                // TODO(AlexeyBarabash): consider to have error code if there will
+                                // be more than only one case of failure
+                                onSyncError(
+                                        getResources()
+                                                .getString(
+                                                        R.string
+                                                                .brave_sync_joining_deleted_account));
```

Comment:
presubmit format likes [Mayakovsky](https://en.wikipedia.org/wiki/Vladimir_Mayakovsky#/media/File:%D0%9A%D0%BE%D0%BB%D1%85%D0%BE%D0%B7%D0%BE%D0%BD%D1%8C_%D1%8D%D1%80%D1%8F%D1%84_1940-06_37%D1%81%D1%82%D1%80.jpg)